pkg/lib/bundle/interfaces.go

```diff
@@ -0,0 +1,21 @@
+package bundle
+
+import (
+	"github.com/operator-framework/operator-registry/pkg/containertools"
+
+	"github.com/sirupsen/logrus"
+)
+
+// BundleImageValidator provides a toolset for pulling and then validating bundle container images
+type BundleImageValidator interface {
+	PullBundleImage(string, string) error
+	ValidateBundle(string) error
```

Comment:
It might be helpful to users of the interface to document the usages of the method parameters either in go doc or by using named parameters.